### ISSUE TITLE
Ability to set validateTrigger on a per-binding basis

### ISSUE THREAD

**Issue by jsobell** [aurelia/validation#303]
```
<input type="text" class="form-control" id="first" placeholder="First Name"
             value.bind="firstName & updateTrigger:'input' & validate">
```
This isn't triggering the validation to occur.
 Example Gist.run: [https://gist.run/?id=21ba7cf3089cf03305749302d9a9f81d](https://gist.run/?id=21ba7cf3089cf03305749302d9a9f81d)

**Comment by jdanyow** [comment]
Currently you can use `updateTrigger: 'blur'` in conjunction with `controller.validateTrigger = validateTrigger.change` but not the other way around. This is a little awkward depending on the number of inputs you want to behave one way or the other.

We can definitely improve this. Adding this issue to the enhancement queue.

**Comment by jsobell** [comment]
Yes, it's a bit tricky. If we can prioritise the updateTrigger over the chosen validateTrigger, that will be great. 

Would making validateTrigger an array of triggers, as validation treats "change" as "change or blur" while the UI doesn't?  Does "bind" make sense as a validateTrigger, so the behavior is always honoured, as this is essentially going to follow exactly the binding.

Also, we need to be able to specify manual validation on at least a per-property basis. If the user has specified manual then none of the triggers should apply to that rule/property. This probably has an overlap with https://github.com/aurelia/validation/issues/301

**Comment by josh-harrison** [comment]
Hi guys, How is the fix supposed to be implemented. Can you tell me what I'm doing wrong here? 

https://gist.run/?id=63e8ae8f65856aef9dc10c6f27292176

The validateTrigger has been set to 'change'. There is an updateTrigger: 'blur' on the name input. However, the validation doesn't occur on blur until something has actually been changed. So if you leave the input in it's initial blank state the required rule won't fire until you type something and then delete it.

Also there is strange behavior on the name input if you change validateTrigger to 'blur'. The input won't validate until it loses focus for the second time. We have use custom attributes like this:

```<input value.bind="customer.dateOfBirth | dateFormat & updateTrigger:'blur' & validate" id="dateOfBirth" type="text" class="form-control" placeholder="Date of birth" />```

And we're seeing that same behavior. Is there a different way we need to implement the custom attribute to prevent this?

**Comment by jdanyow** [comment]
@josh-harrison - check out this section of the docs: http://aurelia.io/hub.html#/doc/article/aurelia/validation/latest/validation-basics/6

(bottom part of the section)
